.github/workflows/build.yml

```diff
@@ -66,6 +66,11 @@ jobs:
         run: |
           set -eux
           python -m pytest -vv -r ap --cov jupyterlab_search_replace
+      
+      - name: Playwright Tests
+        run: |
+          set -eux
+          jlpm playwright test
```

Comment:
You gonna need to create a specific job for it

```suggestion
```

Something like this

```yaml
  integration-tests:
    name: Integration tests
    needs: build
    runs-on: ubuntu-latest

    env:
      PLAYWRIGHT_BROWSERS_PATH: ${{ "{{ github.workspace }}" }}/pw-browsers

    steps:
    - name: Checkout
      uses: actions/checkout@v2

    - name: Base Setup
      uses: jupyterlab/maintainer-tools/.github/actions/base-setup@v1

    - name: Download extension package
      uses: actions/download-artifact@v2
      with:
        name: myextension-sdist
        
    - name: Install the extension
      run: |
        set -eux
        python -m pip install "jupyterlab~=3.1" myextension.tar.gz

    - name: Install dependencies
      working-directory: ui-tests
      env:
        PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: 1
      run: jlpm install
    - name: Set up browser cache
      uses: actions/cache@v2
      with:
        path: |
          ${{ github.workspace }}/pw-browsers
        key: ${{ runner.os }}-${{ hashFiles('ui-tests/yarn.lock') }}
    - name: Install browser
      run: jlpm playwright install chromium
      working-directory: ui-tests

    - name: Execute integration tests
      working-directory: ui-tests
      run: |
        jlpm playwright test

    - name: Upload Playwright Test report
      if: always()
      uses: actions/upload-artifact@v2
      with:
        name: {{ cookiecutter.python_name }}-playwright-tests
        path: |
          ui-tests/test-results
          ui-tests/playwright-report
```

It will also be nice to get a full new file to update the reference snapshots on PR with:

```yaml
name: Update Playwright Snapshots

on:
  issue_comment:
    types: [created, edited]

permissions:
  contents: write
  pull-requests: write

jobs:
  update-snapshots:
    if: ${{ github.event.issue.pull_request && contains(github.event.comment.body, 'please update playwright snapshots') }}
    runs-on: ubuntu-latest

    steps:
      - name: Checkout
        uses: actions/checkout@v2
        with:
          token: ${{ secrets.GITHUB_TOKEN }}

      - name: Configure git to use https
        run: git config --global hub.protocol https

      - name: Checkout the branch from the PR that triggered the job
        run: hub pr checkout ${{ github.event.issue.number }}
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

      - name: Install dependencies
        run: |
          set -eux
          jlpm
          python -m pip install .
      - uses: jupyterlab/maintainer-tools/.github/actions/update-snapshots@v1
        with:
          github_token: ${{ secrets.GITHUB_TOKEN }}
          # Playwright knows how to start JupyterLab server
          start_server_script: 'null'
          test_folder: ui-tests
```